package.json

```diff
@@ -603,7 +604,10 @@
         "antlr4ng": "^3.0.4",
         "await-notify": "1.0.1",
         "d3": "7.9.0",
-        "fs-extra": "11.2.0"
+        "fs-extra": "11.2.0",
+        "node-notifier": "^10.0.1",
+        "swc": "^1.0.11",
+        "@swc/core": "^1.7.14"
```

Comment:
What are these 3 node packages for? I don't see anything in this patch which uses any of them.

Reply:
`npm ls -all`

listed them as optional dependencies.

Reply:
We don't need them so please remove that.